pkgs/by-name/fi/fixjson/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub
+}:
+
+buildNpmPackage {
+  pname = "fixjson";
+  version = "1.1.2";
```

```suggestion
  version = "1.1.2-unstable-2021-01-05";
```

Since upstream has no tags...

---

Done!